hscontrol/policy/v2/types.go

```diff
@@ -922,12 +958,12 @@ type Policy struct {
 }
 
 var (
-	autogroupForSrc       = []AutoGroup{}
-	autogroupForDst       = []AutoGroup{AutoGroupInternet}
-	autogroupForSSHSrc    = []AutoGroup{}
-	autogroupForSSHDst    = []AutoGroup{}
+	autogroupForSrc       = []AutoGroup{AutoGroupMember, AutoGroupTagged}
```

Comment:
Add a TODO for me here to add these kind of checks for tagOwners and autoApprovers too.